src/azure-cli-core/azure/cli/core/extension/__init__.py

```diff
@@ -22,7 +22,7 @@
 EXTENSIONS_DIR = os.path.expanduser(_CUSTOM_EXT_DIR) if _CUSTOM_EXT_DIR else os.path.join(GLOBAL_CONFIG_DIR,
                                                                                           'cliextensions')
 DEV_EXTENSION_SOURCES = _DEV_EXTENSION_SOURCES.split(',') if _DEV_EXTENSION_SOURCES else []
-EXTENSIONS_SYS_DIR = os.path.expanduser(_CUSTOM_EXT_SYS_DIR) if _CUSTOM_EXT_SYS_DIR else os.path.join(get_python_lib(), 'azure-cli-extensions')
+EXTENSIONS_SYS_DIR = os.path.expanduser(_CUSTOM_EXT_SYS_DIR) if _CUSTOM_EXT_SYS_DIR else os.path.join(get_path("platlib"), 'azure-cli-extensions')
```

Comment:
> distutils.sysconfig.get_python_lib([plat_specific[, standard_lib[, prefix]]])
Return the directory for either the general or platform-dependent library installation. If plat_specific is true, the platform-dependent include directory is returned; if false or omitted, the platform-independent directory is returned. --https://docs.python.org/3.11/distutils/apiref.html#distutils.sysconfig.get_python_lib

Since it returns the **platform-independent directory**, we should use `get_path('purepath')`.
> platlib: directory for site-specific, platform-specific files.
purelib: directory for site-specific, non-platform-specific files (‘pure’ Python).
--https://docs.python.org/3/library/sysconfig.html#sysconfig.get_platform

Previous, it was discussed in https://github.com/Azure/azure-cli/pull/20391#issuecomment-972426138

Reply:
Alright.

Reply:
Sorry, I have a typo in my previous comment. It should be `purelib`